keras/utils/vis_utils.py

```diff
@@ -102,7 +102,6 @@ def model_to_dot(model,
 
     # Connect nodes with edges.
     for layer in layers:
-        layer_id = str(id(layer))
```

Comment:
Sounds it would be better to remove the definition on line 110, not this one?

Reply:
You are right! My bad.